cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -313,7 +313,7 @@ func (s *CMServer) Run(_ []string) error {
 	// If apiserver is not running we should wait for some time and fail only then. This is particularly
 	// important when we start apiserver and controller manager at the same time.
 	var versionStrings []string
-	err = wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
+	err = wait.Poll(time.Second, 10*time.Second, func() (bool, error) {
 		versionStrings, err = client.ServerAPIVersions(kubeconfig)
 		return err == nil, err
```

Comment:
@lavalamp Why mask the error if there is an error? Shouldn't it end up returning ```True```,```nil``` or ```False```, ```ErrorInstance```?